tasks/main.yml

```diff
@@ -27,6 +27,6 @@
     src: "{{ dotfiles_repo_local_destination }}/{{ item }}"
     dest: "{{ dotfiles_home }}/{{ item }}"
     state: link
-    mode: 0644
+    mode: "u=rwX,g=rX,o=rX"
```

Comment:
`mode: 0755` would also be possible.

Reply:
`0755`= `u=rwX,g=rX,o=rX`

If you want different permissions for files and directories you have to check you content first.

Reply:
No, `0755` equals to `u=rwx,g=rx,o=rx`. There is a difference between `x` and `X`:

-  `x` sets the execute permission unconditionally
-  `X` set the execute permission only on directories and files that already have the execute permission for some user.

Source: `man chmod`

Reply:
Thank you! I didn't know that.